Dockerfile

```diff
@@ -28,9 +27,7 @@ RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.
     curl -s -L https://github.com/CycloneDX/sbom-utility/releases/download/v"${sbom_utility_version}"/sbom-utility-v"${sbom_utility_version}"-linux-amd64.tar.gz --output sbom-utility.tar.gz && \
```

Comment:
Does this line also have to be removed ?

Reply:
no, it's sbom-utility and that's still in the use

Reply:
https://github.com/redhat-appstudio/build-definitions/blob/54ecd81ce2791a83da05e289bd7630b87e201d55/task/sbom-json-check/0.1/sbom-json-check.yaml#L57